generators/python/scripts/text_to_speech.py

```diff
@@ -0,0 +1,19 @@
+import rospy
```

Comment:
let's leave it for code completion. There is no harm from it but it will allow to catch misspelled functions from module rospy.